MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.TabControl.xaml

```diff
@@ -261,57 +274,39 @@
                             <Setter TargetName="SelectionHighlightBorder" Property="Visibility" Value="Visible" />
                             <Setter TargetName="contentPresenter" Property="Foreground" Value="{DynamicResource PrimaryHueMidBrush}"/>
                         </MultiTrigger>
-                        <Trigger Property="wpf:ColorZoneAssist.Mode" Value="PrimaryMid">
```

Comment:
Any reason for the removal of these triggers? Doesn't this cause issues for anyone setting the Mode attached property?